spring-kafka/src/main/java/org/springframework/kafka/listener/AbstractConsumerSeekAware.java

```diff
@@ -33,15 +34,23 @@
  * having to keep track of the callbacks itself.
  *
  * @author Gary Russell
+ * @author Borahm Lee
  * @since 2.3
  *
  */
 public abstract class AbstractConsumerSeekAware implements ConsumerSeekAware {
 
 	private final Map<Thread, ConsumerSeekCallback> callbackForThread = new ConcurrentHashMap<>();
 
+	/*
+	* @deprecated Replaced by the {@link #topicToCallbacks}.
+	* see https://github.com/spring-projects/spring-kafka/pull/3341
+	*/
+	@Deprecated(since = "3.3", forRemoval = true)
```

Comment:
There is no reason in deprecating `private` props: it is totally OK to rework internals to a new state.